src/sudoku/singlarobin/react/component/Input/index.js

```diff
@@ -0,0 +1,31 @@
+import classes from './styles.module.css';
+
+const Input = (props) => {
+    const { index, value, handleInputChange } = props;
+
+    const handleChange = (e) => {
+        let sudoku = JSON.parse(localStorage.getItem('sudoku'));
+        if (e.target.value !== '') {
+            sudoku[2]['undo'].push({ 'index': index, 'prev': value, 'current': e.target.value });
```

Comment:
'index': index -> { index, ...

Reply:
{ index, 'prev': value, 'current': e.target.value }

In this way I have to write??